From c7b41da6d116a1da4edb41c95a31d45d74c813ab Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Tue, 7 Aug 2018 17:22:06 -0500
Subject: [PATCH] remove hardcoded make

---
 lapack-netlib/make.inc.example | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/lapack-netlib/make.inc.example b/lapack-netlib/make.inc.example
index 57fd51e..545a4bc 100644
--- a/lapack-netlib/make.inc.example
+++ b/lapack-netlib/make.inc.example
@@ -8,8 +8,8 @@ SHELL = /bin/sh
 
 #  CC is the C compiler, normally invoked with options CFLAGS.
 #
-CC = gcc
-CFLAGS = -O3
+# CC = gcc
+# CFLAGS = -O3
 
 #  Modify the FC and FFLAGS definitions to the desired compiler
 #  and desired compiler options for your machine.  NOOPT refers to
@@ -19,10 +19,10 @@ CFLAGS = -O3
 #  and handle these quantities appropriately. As a consequence, one
 #  should not compile LAPACK with flags such as -ffpe-trap=overflow.
 #
-FC = gfortran
-FFLAGS = -O2 -frecursive
-FFLAGS_DRV = $(FFLAGS)
-FFLAGS_NOOPT = -O0 -frecursive
+# FC = gfortran
+# FFLAGS = -O2 -frecursive
+# FFLAGS_DRV = $(FFLAGS)
+# FFLAGS_NOOPT = -O0 -frecursive
 
 #  Define LDFLAGS to the desired linker options for your machine.
 #
@@ -31,9 +31,9 @@ LDFLAGS =
 #  The archiver and the flag(s) to use when building an archive
 #  (library).  If your system has no ranlib, set RANLIB = echo.
 #
-AR = ar
-ARFLAGS = cr
-RANLIB = ranlib
+# AR = ar
+# ARFLAGS = cr
+# RANLIB = ranlib
 
 #  Timer for the SECOND and DSECND routines
 #
