diff --git a/utest/CMakeLists.txt b/utest/CMakeLists.txt
index 0c99e0d1..a69e1b14 100644
--- a/utest/CMakeLists.txt
+++ b/utest/CMakeLists.txt
@@ -28,12 +28,6 @@ endif ()
 # known to hang with the native Windows and Android threads
 # FIXME needs checking if this works on any of the other platforms
 if (OS_CYGWIN_NT OR OS_LINUX)
-if (NOT USE_OPENMP)
-set(OpenBLAS_utest_src
-  ${OpenBLAS_utest_src}
-  test_fork.c
-  )
-endif()
 set(OpenBLAS_utest_src
   ${OpenBLAS_utest_src}
   test_post_fork.c
diff --git a/utest/Makefile b/utest/Makefile
index fad3607d..22ef4ba6 100644
--- a/utest/Makefile
+++ b/utest/Makefile
@@ -26,9 +26,6 @@ endif
 #this does not work with OpenMP nor with native Windows or Android threads
 # FIXME TBD if this works on OSX, SunOS, POWER and zarch
 ifeq ($(OSNAME), $(filter $(OSNAME),Linux CYGWIN_NT))
-ifneq ($(USE_OPENMP), 1)
-OBJS += test_fork.o
-endif
 OBJS += test_post_fork.o
 endif

